contrib/update-lock-files.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+#
+# Update the minimal/recent lock file
+
+set -euo pipefail
+
+for file in Cargo-minimal.lock Cargo-recent.lock; do
+    cp --force "$file" Cargo.lock
+    cargo check  --color=always --workspace --all-features --all-targets
```

Comment:
I think we need cargo check --tests

Reply:
AFAIK `--all-targets` includes examples, benches, and tests

Reply:
If the goal is just to update the lockfile do we actually need any of these flags?

I'm fine to keep them but in practice I always ctrl-C the `cargo check` invocation as soon as I see any output, because at that point it's already updated the lockfile.

Reply:
Ha, we definitely don't need `--color-always`, I also ctrl-C soon as it starts scrolling past. (I just blindly cut n paste my shell alias when I wrote this.)